tools/clang/test/HLSL/string.hlsl

```diff
@@ -50,7 +50,7 @@ matrix<string, 4, 4> g_strMatrix1;                          /* expected-error {{
 void hello_here(string message, string s, float f) {        /* expected-error {{parameter of type string is not supported}} expected-error {{parameter of type string is not supported}} fxc-pass {{}} */
   printf(s);
   printf(message);
-  printf("%f", f);                                          /* expected-error {{use of undeclared identifier 'printf'}} fxc-pass {{}} */
```

Comment:
Is this necessary? I suspect it might be because some fool ported a bunch of DXIL tests to Linux? It further makes me wonder if there might be utility in adding a constant define that indicates if the current compile is targeting SPIR-V. I don't think such a thing exists now. Does it?